code/datums/mind.dm

```diff
@@ -119,6 +119,18 @@
 			output += "<br><b>Your [A.role_text] objectives:</b>"
 		output += "[A.print_objectives(FALSE)]"
 
+	if(LAZYLEN(individual_objectives))
+		output += "<HR><B>Your individual objectives:</B><UL>"
+		var/obj_count = 1
+		var/la_explanation
+		for(var/datum/individual_objective/objective in individual_objectives)
+			output += "<br><b>#[obj_count] [objective.name][objective.limited_antag ? " [objective.show_la]" : ""]</B>: [objective.get_description()]</b>"
+			obj_count++
+			if(objective.limited_antag)
+				la_explanation = objective.la_explanation
+		output += "</UL>"
+		if(la_explanation)
+			output += la_explanation
```

Comment:
It would be nice to condense both code blocks into one new function to reduce clutter